components/footer/_footer2.scss

```diff
@@ -0,0 +1,38 @@
+.footer2 {
```

Comment:
.footer2 needs renaming. Is this replacing the old footer?

Reply:
This is the service-wide public facing footer, there will eventually be a slimmer version for within the ‘transaction funnel’ of eQ and a different variant for Internal UI.